mappings/net/minecraft/world/biome/source/util/MultiNoiseUtil.mapping

```diff
@@ -0,0 +1,123 @@
+		ARG 3 erosion
+		ARG 4 depth
+		ARG 5 weirdness
+		ARG 6 offset
+	METHOD method_37625 createMixedNoisePoint (Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;F)Lnet/minecraft/class_6452$class_4762;
+		ARG 0 temperature
+		ARG 1 humidity
+		ARG 2 continentalness
+		ARG 3 erosion
+		ARG 4 depth
+		ARG 5 weirdness
+		ARG 6 offset
+	CLASS class_4762 MixedNoisePoint
+		COMMENT Represents a point in a multi-dimensional cartesian plane. Mixed-noise
+		COMMENT biome generator picks the closest noise point from its selected point
+		COMMENT and choose the biome associated to that closest point. Another factor,
+		COMMENT rarity potential, favors larger differences in values instead, contrary
+		COMMENT to other point values.
+		FIELD field_22043 temperature Lnet/minecraft/class_6452$class_6454;
+		FIELD field_22044 humidity Lnet/minecraft/class_6452$class_6454;
+		FIELD field_22046 weirdness Lnet/minecraft/class_6452$class_6454;
+		FIELD field_22047 offset F
+			COMMENT This value awards another point with value farthest from this one; i.e.
+			COMMENT unlike other points where closer distance is better, for this value the
+			COMMENT farther the better. The result of the different values can be
+			COMMENT approximately modeled by a hyperbola weight=cosh(peak-1) as used by the
+			COMMENT mixed-noise generator.
+		FIELD field_24679 CODEC Lcom/mojang/serialization/Codec;
+		FIELD field_34174 continentalness Lnet/minecraft/class_6452$class_6454;
+		FIELD field_34175 erosion Lnet/minecraft/class_6452$class_6454;
+		FIELD field_34176 depth Lnet/minecraft/class_6452$class_6454;
+		FIELD field_34177 parameters Ljava/util/List;
+		METHOD <init> (Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;F)V
+			ARG 1 temperature
+			ARG 2 humidity
+			ARG 3 continentalness
+			ARG 4 erosion
+			ARG 5 depth
+			ARG 6 weirdness
+			ARG 7 offset
+		METHOD equals (Ljava/lang/Object;)Z
+			ARG 1 o
+		METHOD method_37642 getTemperature ()Lnet/minecraft/class_6452$class_6454;
+		METHOD method_37643 (Lnet/minecraft/class_6452$class_4762;)F
+			ARG 1 point
+		METHOD method_37644 (Lnet/minecraft/class_6452$class_6460;)F
+			ARG 1 point
+		METHOD method_37646 getHumidity ()Lnet/minecraft/class_6452$class_6454;
+		METHOD method_37648 getContinentalness ()Lnet/minecraft/class_6452$class_6454;
+		METHOD method_37650 getErosion ()Lnet/minecraft/class_6452$class_6454;
+		METHOD method_37652 getDepth ()Lnet/minecraft/class_6452$class_6454;
+		METHOD method_37654 getWeirdness ()Lnet/minecraft/class_6452$class_6454;
+		METHOD method_37656 getOffset ()F
+		METHOD method_37658 getParameters ()Ljava/util/List;
+	CLASS class_6454 ParameterConfig
```

Comment:
This should be a min and a max range. The methods get the distance from a point to the range or from another range to the range, though the code isn't straightforward up front

Reply:
I looked at `method_37627` when mapping it, and I thought that it may be a range, but I wasn't sure. I'll rename that.